thanks-bot/bot.js

```diff
@@ -1,35 +1,57 @@
-var Discord = require('discord.io');
-var logger = require('winston');
-var auth = require('./auth.json');
+const Discord = require('discord.js')
+const logger = require('winston')
+const auth = require('../auth.json')
 
 // Configure logger settings
-logger.remove(logger.transports.Console);
+logger.remove(logger.transports.Console)
 logger.add(logger.transports.Console, {
-    colorize: true
-});
-
-logger.level = 'debug';
+  colorize: true
+})
+logger.level = 'debug'
 
 // Initialize Discord Bot
-var bot = new Discord.Client({
-   token: auth.token,
-   autorun: true,
-});
+const bot = new Discord.Client()
 
 bot.on('ready', function (evt) {
-    logger.info('Connected');
-    logger.info('Logged in as: ');
-    logger.info(bot.username + ' - (' + bot.id + ')');
-});
-
-bot.on('message', function (user, userID, channelID, message, evt) {
-    switch(message) {
-        case 'Ассаляму аляйкум':
-            bot.sendMessage({
-                to: channelID,
-                message: 'Уа аляйкум салям',
-            });
-
-            break;
-    }
-});
+  logger.info('Connected')
```

Comment:
Let's do the same thing as in Buttler Bot now? Can you check out the code there and copy it here?